packages/db/tests/models/test-base-model.js

```diff
@@ -137,3 +146,47 @@ test('BasePgModel.exists() correctly returns false', async (t) => {
   const info = cryptoRandomString({ length: 5 });
   t.false(await basePgModel.exists(knex, { info }));
 });
+
+test('BasePgModel.delete() correctly deletes records', async (t) => {
+  const { knex, basePgModel, tableName } = t.context;
+  const info = cryptoRandomString({ length: 5 });
+
+  // Insert the record and validate that it exists in the table
+  const [recordCumulusId] = await knex(tableName)
+    .insert({ info })
+    .returning('cumulus_id');
+
+  t.true(
+    await basePgModel.exists(knex, { cumulus_id: recordCumulusId })
+  );
+
+  // Delete the record and validate that it's gone
+  t.is(
+    await basePgModel.delete(knex, { cumulus_id: recordCumulusId }),
+    1
+  );
+
+  await t.throwsAsync(
+    basePgModel.get(knex, { cumulus_id: recordCumulusId }),
+    { instanceOf: RecordDoesNotExist }
+  );
+});
+
+test('BasePgModel.delete() works with knex transaction', async (t) => {
+  const { knex, basePgModel, tableName } = t.context;
+  const info = cryptoRandomString({ length: 5 });
+
+  const [recordCumulusId] = await knex(tableName)
+    .insert({ info })
+    .returning('cumulus_id');
+
+  t.is(await knex.transaction(
+    (trx) => basePgModel.delete(trx, { cumulus_id: recordCumulusId })
+  ), 1);
+
+  // validate that the record is not in the table
+  await t.throwsAsync(
```

Comment:
Use the `exists` method

Reply:
👍 5755cf6849bb12d81e3638239d53b38bee66d69f